### ISSUE TITLE
Compatibility for newer versions of TensorFlow

### ISSUE THREAD

**Issue by henryre** [HazyResearch/tanda#19]
As evidenced by #16 and #18, TANDA doesn't currently work correctly with the newest versions of TensorFlow. We developed with TF 1.2, and we know it works with that. But haven't confirmed our results with anything newer. We believe the primary issue is the application of batch norm, but there may have been other subtle op changes.

**Comment by caterpillars** [comment]
Hi, 

I am trying to install tensorflow 1.2, but it is not in the available versions (1.13.0rc1, 1.13.0rc2, 1.13.1, 1.13.2, 1.14.0rc0, 1.14.0rc1, 1.14.0, 1.15.0rc0, 1.15.0rc1, 2.0.0a0, 2.0.0b0, 2.0.0b1, 2.0.0rc0, 2.0.0rc1, 2.0.0rc2). In this case, should I use the latest one before 1.2 (1.15.0rc1) instead?

Thanks,

Hongli
